src/lib/model.ts

```diff
@@ -35,11 +35,13 @@ export interface Organization extends Entity {
   name: string;
   avatar: string;
   description: string;
+  teams: Team[];
 }
 
 export interface Event extends Entity {
   type: string;
   comment: string;
+  createdFor: string;
```

Comment:
`createdFor` field was assumed to reference the user, associated with the event. Thus, changes made in the event modal dialog could easily be sent to the server.